src/lint/concept_exercises.nim

```diff
@@ -0,0 +1,37 @@
+import std/[json, os, terminal]
+import ".."/helpers
+import "."/validators
+
+proc isValidAuthorOrContributor(data: JsonNode, key: string, path: string): bool =
+  checkObject(key)
+  checkStringKey("github_username", isRequired = true)
+  checkStringKey("exercism_username", isRequired = false)
+
+template isValidFiles(data: JsonNode, context, path: string) =
+  checkObject(context)
+  checkArrayOfStrings(context, "solution", isRequired = true)
+  checkArrayOfStrings(context, "test", isRequired = true)
+  checkArrayOfStrings(context, "exemplar", isRequired = true)
+
+proc isValidConceptExerciseConfig(data: JsonNode, path: string): bool =
+  checkObject("root")
+  checkArrayOf("authors", isValidAuthorOrContributor, isRequired = true)
+  checkArrayOf("contributors", isValidAuthorOrContributor, isRequired = false)
+  checkArrayOfStrings("", "forked_from", isRequired = false)
+  isValidFiles(data, "files", path)
+  checkStringKey("language_versions", isRequired = false)
```

Comment:
I really like this approach. It is very readable.

Reply:
I think it's better now with `isRequired = true` by default. Do you agree?

Reply:
Yes that is fine.